mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -361,12 +361,28 @@ func TestAccContainerCluster_withAuthenticatorGroupsConfig(t *testing.T) {
 		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccContainerCluster_withAuthenticatorGroupsConfig(containerNetName, clusterName, orgDomain),
+				Config: testAccContainerCluster_withAuthenticatorGroupsConfig(containerNetName, clusterName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckNoResourceAttr("google_container_cluster.with_authenticator_groups",
+						"authenticator_groups_config.0.enabled"),
+				),
 			},
 			{
-				ResourceName:        "google_container_cluster.with_authenticator_groups",
-				ImportState:         true,
-				ImportStateVerify:   true,
+				ResourceName:      "google_container_cluster.with_authenticator_groups",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccContainerCluster_withAuthenticatorGroupsConfigUpdate(containerNetName, clusterName, orgDomain),
```

Comment:
Can we test in the other direction, unsetting the field as well?

Reply:
updated